types/node/os.d.ts

```diff
@@ -172,6 +172,64 @@ declare module "os" {
             ETXTBSY: number;
             EWOULDBLOCK: number;
             EXDEV: number;
+            WSAEINTR: number;
```

Comment:
Nit: Can we sort those please?

Reply:
I sorted this list as this sorted at [source code](https://github.com/nodejs/node/blob/902c9fac194d06aec5fd5307a6712c39a6da18a7/src/node_constants.cc#L379).